python/feathub/feature_views/derived_feature_view.py

```diff
@@ -47,7 +50,10 @@ def __init__(
                          feature is a string, it should be either in the format
                          {table_name}.{feature_name}, which refers to a feature in the
                          table with the given name, or in the format {feature_name},
-                         which refers to a feature in the source table. For any feature
+                         which refers to a feature in the source table. If
```

Comment:
Even if we don't add this paragraph in the doc, the existing doc already says that we will compute these `features`. So the doc seems redundant here, right?

Should we update the doc for `keep_source_fields`?